tests/Unit/Command/SendInvoiceEmailTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\InvoicingPlugin\Unit\Command;
+
+use PHPUnit\Framework\TestCase;
+use Sylius\InvoicingPlugin\Command\SendInvoiceEmail;
+
+final class SendInvoiceEmailTest extends TestCase
+{
+    /** @test */
```

Comment:
Use attribute instead of annotation
```suggestion
    #[Test]
```

Reply:
done